tests/manage/pv_services/test_resource_deletion_during_pvc_pod_creation_and_io.py

```diff
@@ -4,127 +4,213 @@
 
 from ocs_ci.framework.testlib import ManageTest, tier4
 from ocs_ci.ocs import constants
-from ocs_ci.ocs.resources.pvc import delete_pvcs
 from ocs_ci.ocs.resources.pod import (
-    get_fio_rw_iops, get_mds_pods, get_mon_pods, get_mgr_pods, get_osd_pods
+    get_mds_pods, get_mon_pods, get_mgr_pods, get_osd_pods
 )
+from ocs_ci.utility.utils import TimeoutSampler
 from tests import helpers, disruption_helpers
-from tests.fixtures import (
-    create_rbd_storageclass, create_ceph_block_pool,
-    create_cephfs_storageclass, create_rbd_secret, create_cephfs_secret,
-    create_project, create_pvcs
-)
-from ocs_ci.utility.utils import run_cmd
 
 log = logging.getLogger(__name__)
 
 
-@pytest.fixture()
-def test_fixture(request):
+@tier4
+@pytest.mark.parametrize(
+    argnames=['interface', 'resource_to_delete'],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-735")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'mon'],
+            marks=pytest.mark.polarion_id("OCS-736")
+        ),
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, 'osd'],
+            marks=pytest.mark.polarion_id("OCS-737")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'mgr'],
+            marks=pytest.mark.polarion_id("OCS-738")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'mon'],
+            marks=pytest.mark.polarion_id("OCS-739")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'osd'],
+            marks=pytest.mark.polarion_id("OCS-740")
+        ),
+        pytest.param(
+            *[constants.CEPHFILESYSTEM, 'mds'],
+            marks=pytest.mark.polarion_id("OCS-741")
+        )
+
+    ]
+)
+class TestResourceDeletionDuringCreationOperations(ManageTest):
     """
-    Setup and teardown
+    Test class
```

Comment:
Please add more details to the docstring of this class

Reply:
Done